### ISSUE TITLE
[Event Request] Codeunit 5895 "Inventory Adjustment".InitializeAdjmt

### ISSUE THREAD

**Issue by fridrichovsky** [microsoft/ALAppExtensions#22191]
Please add new event

```
local procedure InitializeAdjmt()
begin
    Clear(LevelNo);
    MaxLevels := 100;
    WindowUpdateDateTime := CurrentDateTime;
    if not IsOnlineAdjmt then
        OpenWindow();

    Clear(ItemJnlPostLine);
    ItemJnlPostLine.SetCalledFromAdjustment(true, PostToGL);

    InvtSetup.Get();
    GLSetup.Get();
    PostingDateForClosedPeriod := GLSetup.FirstAllowedPostingDate();
    //------------------------------------------------------------OnInitializeAdjmtOnAfterGetPostingDate:BEGIN
    OnInitializeAdjmtOnAfterGetPostingDate(PostingDateForClosedPeriod);
    //------------------------------------------------------------OnInitializeAdjmtOnAfterGetPostingDate:END        
    GetAddReportingCurrency();

    SourceCodeSetup.Get();

    ItemCostMgt.SetProperties(true, 0);
    TempJobToAdjustBuf.DeleteAll();
end;

//------------------------------------------------------------OnInitializeAdjmtOnAfterGetPostingDate:BEGIN
[IntegrationEvent(true, false)]
local procedure OnInitializeAdjmtOnAfterGetPostingDate(var _PostingDateForClosedPeriod: Date)
begin
end
//------------------------------------------------------------OnInitializeAdjmtOnAfterGetPostingDate:END        

```

**Comment by fridrichovsky** [comment]
Hello @AlexanderYakunin can you process this issue?

**Comment by fridrichovsky** [comment]
Hello @AlexanderYakunin can you process this issue?

**Comment by AlexanderYakunin** [comment]
Already implemented in 22.2

    [IntegrationEvent(false, false)]
    local procedure OnInitializeAdjmtOnAfterGetPostingDate(var PostingDateForClosedPeriod: Date)
    begin
    end;
